pkg/routine/recurringtask.go

```diff
@@ -46,32 +45,34 @@ func NewRecurringTask(t Task, i time.Duration, ops ...RecurringTaskOption) *Recu
 }
 
 // Start starts the timer
-func (t *RecurringTask) Start(ctx context.Context) error {
+func (t *RecurringTask) Start(_ context.Context) error {
 	t.ticker = t.clock.Ticker(t.interval)
+	t.done = make(chan struct{})
```

Comment:
move to line 39 above

Reply:
It is useful in `TestRecurringTaskStopBeforeStart`